test/repository.spec.js

```diff
@@ -349,11 +349,12 @@ describe('Repository', function() {
 
       it('should write to repo', function(done) {
          remoteRepo.writeFile('master', fileName, initialText, initialMessage, assertSuccessful(done, function() {
-            remoteRepo.getContents('master', fileName, 'raw', assertSuccessful(done, function(err, fileText) {
+            wait().then(() => remoteRepo.getContents('master', fileName, 'raw',
```

Comment:
Blind wait's a generally very brittle. Can you pull github for the file and if it doesn't show up within 20 seconds then fail?